tabby-core/src/services/fileProviders.service.ts

```diff
@@ -13,8 +13,9 @@ export class FileProvidersService {
     ) { }
 
     async selectAndStoreFile (description: string): Promise<string> {
-        const p = await this.selectProvider()
-        return p.selectAndStoreFile(description)
+        return this.selectProvider().then(p => {
```

Comment:
no behaviour changes here?

Reply:
It should not, I didn't notice any but my understanding could be wrong.
I tried to avoid `Uncaught error in Promise` cause by awaiting `selectProvider`. Chaining promise here allow me to return a rejected promise if `selectProvider` is rejected too.